src/providers/app-config/__mocks__/app-config.mock.ts

```diff
@@ -12,6 +12,10 @@ export class AppConfigProviderMock {
     return;
   }
 
+  public getPersonalJournalUrl(): string {
+    return 'https://www.example.com/api/v1/journals/1234/personal'
+  }
```

Comment:
Could we make the mock id 8 digits so it matches the data we have in Active Directory

Reply:
Done.